backend/src/models/encounter.model.ts

```diff
@@ -12,7 +12,9 @@ export interface EncounterModel {
 const schema = new Schema<EncounterModel>({
   title: { type: String, required: true },
   date: { type: Date, required: false },
-  time_updated: { type: Date, default: new Date(Date.now()), required: true },
+  time_updated: {
+    type: Date, default: new Date(Date.now()), expires: 31536000, required: true, // Added a one year expiry time to Encounter entries
+  },
```

Comment:
Hmm I'm not sure why we need an expiry date?

Reply:
In the [#234]  issue description it said "there will be a maximum time where all encounters will be deleted to prevent bloat", so I just added a 1 year expiry. Can change it if needed

Reply:
Ohh right I didnt see that part. In that case LGTM!